benchmarks/index.js

```diff
@@ -43,6 +43,7 @@ function HyperbahnBenchmarkRunner(opts) {
     }
 
     var self = this;
+    opts.torchDelay = opts.torchDelay || 15 * 1000;
```

Comment:
anyhow, if we keep it please:
- make the units be seconds in like `torchTime`
- add the default to `torch.json` for affordance